### PR TITLE
[NUI][TCSACR-485] Add some methods for Loading

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- Make Play / Pause / Stop methods to public API on API level 10.
- These methods are for Loading animation behavior.

### API Changes ###
 - ACR: TCSACR-485
   https://code.sec.samsung.net/jira/browse/TCSACR-483

Added:
 - public void Play()
 - public void Pause()
 - public void Stop()

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/Loading.cs [hunk 1]
```diff
@@ -339,6 +336,7 @@ public void Stop()
 
         private void Initialize()
         {
+            Focusable = true;
```
Comment (myroot) [inline src/Tizen.NUI.Components/Controls/Loading.cs:339]: 포커스를 갖을 필요가 있나요?

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Loading.cs:339]: 아 넵 해당 라인 지워서 다시 업데이트하겠습니다.
테스트하며 같이 올라갔습니다 :(
리뷰 감사합니다~

**Review by myroot — APPROVED** [review]

**Comment by dongsug-song** [comment]
the ACR has been accepted, so this is merged now

File: src/Tizen.NUI.Components/Controls/Loading.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿/*
- * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
```
Comment (wiertel) [inline src/Tizen.NUI.Components/Controls/Loading.cs:2]: If it's not a new file, then it would be better to put list or range:  "2021, 2022" OR "2021-2022"

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/Loading.cs:2]: Thanks for reviewing this PR.
Yes, it's not a new file. However, as you know, NUI is csharp-binding of native DALi, and DALi has a rule that updates the copyright year when the file is updated. So, as far as I know, NUI also follows this rule.

**Pull request merged**